api/app/Models/Department.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+/**
+ * Class Department
+ *
+ * @property int $id
+ * @property int department_number
```

Comment:
The [department csv](http://donnees-data.tpsgc-pwgsc.gc.ca/ba1/min-dept/min-dept.csv) uses leadings zeros, for example `"006","Finance (Department of)","Finances (Ministère des)"`. Is the plan to pad these in later for if and when they need to be displayed?

Reply:
Ya I believe that's the easiest way (if we plan to display them like that).